modules/ossm-deploy-multi-mesh.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deploy-mod-v2x.adoc
+
+
+[id="ossm-deploy-multi-mesh_{context}"]
+= Multimesh or Federated deployment model
+
+[role="_abstract"]
+*Federation* is a deployment model that lets you share services and workloads between separate meshes managed in distinct administrative domains.
+
+The Istio multi-cluster model requires a high level of trust between meshes, and requires remote access to all Kubernetes API servers on which the individual meshes reside. {ProductName} federation takes an opinionated approach to a multi-cluster implementation of Service Mesh that assumes _minimal_ trust between meshes.
+
+A *federated mesh* is a group of meshes behaving as a single mesh. The services in each mesh may be unique services (that is, adding services to the mesh importing them), may provide additional workloads for the same services across the meshes (for example, providing high availability), or a combination of both. All meshes that are joined into a federated mesh remain managed individually, and you must explicitly configure which services are exported to and imported from other meshes in the federation. Support functions such as certificate generation, metrics and trace collection remain local in their respective meshes.
```

Comment:
The explanatory note in brackets seems a bit confusing - (that is, adding services to the mesh importing them). 

The meaning might be clear but 2 gerunds somewhat hide who or what is adding and importing what. Is it possible to rephrase it with that- clauses maybe?
And "that is" is somewhat colloquial?

I also had a feeling we avoid providing additional information and examples inside brackets, but maybe it's just a guideline for the sets of docs I work with, we're expected to use commas or add separate sentences.